### DISCUSSION TITLE
next.config.js: Redirect with multiple sources

### DISCUSSION THREAD

**Discussion by 5ervant** [vercel/next.js#22531] [Ideas]
next.config.js: Redirects | Next.js: https://nextjs.org/docs/api-reference/next.config.js/redirects

Is there any redirection property for `next.config.js` that accepts multiple sources with 1 destination like the following?

`````
module.exports = {
  async redirects() {
    return [
      {
        source: ['/source1', '/source2'],  // Should be string.
        destination: '/',
        permanent: true,
      },
    ]
  },
}
`````

I'm just getting source isn't string upon putting an array value into `source`.

**Comment by Dragate** [comment]
Take a look at the solution here: https://github.com/vercel/next.js/discussions/22401
